sys-libs/libselinux/libselinux-3.8.1.ebuild

```diff
@@ -147,6 +131,29 @@ multilib_src_install() {
 	use static-libs || rm "${ED}"/usr/$(get_libdir)/*.a || die
 }
 
+multilib_src_install_all() {
+	if use python; then
+		cd src || die
+		mv selinux.py __init__.py || die
+		distutils-r1_src_install
+	fi
+}
+
+python_install() {
+	# this installs the C extensions only
+	distutils-r1_python_install
+
+	# now explicitly install the python package
+	python_moduleinto selinux
+	python_domodule __init__.py
+
+	# install the C extension symlink
+	local pycext="$(python -c 'import importlib.machinery;print(importlib.machinery.EXTENSION_SUFFIXES[0])')"
+	dosym -r "$(python_get_sitedir)/selinux/_selinux${pycext}" "$(python_get_sitedir)/_selinux${pycext}"
+
+	python_optimize # bug 531638
```

Comment:
```suggestion
```

python_domodule has already done it for selinux and I suppose there are no other modules.